iron-image.html

```diff
@@ -105,9 +114,11 @@
 
   <template>
 
-    <img id="img" role="none" hidden$="[[_computeImageVisibility(sizing)]]">
-    <div id="placeholder" hidden$="[[_computePlaceholderVisibility(fade,loaded,preload)]]" class$="[[_computePlaceholderClassName(fade,loaded,preload)]]"></div>
-    <content></content>
+    <div id="backdrop" hidden$="[[_computeBackdropHidden(sizing)]]"></div>
+    <img id="img" role="none" hidden$="[[_computeImgHidden(sizing)]]">
```

Comment:
I'm not sure that `role="none"` is a thing. Is it a thing?

I think what we want here is to use `aria-hidden="true"`. http://www.w3.org/TR/wai-aria/states_and_properties#aria-hidden